packages/oui-list/_mixins.less

```diff
@@ -197,7 +197,7 @@
       border-bottom: @separator-size solid @separator-color;
 
       &:last-child {
-          border-bottom: 0 none;
+        border-bottom: 0 none;
```

Comment:
:tada: Conjuring with this: https://github.com/ovh-ux/stylelint-config-ovh, it will be awesome :unicorn: !